docs/breadcrumb/toc.yml

```diff
@@ -464,7 +464,7 @@ items:
           topicHref: /dotnet/csharp/language-reference/proposals/csharp-8.0/index
         - name: C# 9.0 feature specifications
           tocHref: /dotnet/csharp/language-reference/proposals/csharp-9.0/
-          topicHref: /dotnet/csharp/language-reference/proposals/csharp-9.0/index
+          topicHref: /dotnet/csharp/language-reference/proposals/csharp-9.0/records
```

Comment:
@BillWagner I see that for C# 8, this is done through a redirection entry instead. LMK if you'd prefer that here.

Reply:
@gewarren The reason I added the redirect for "index.md" was so hackable URLs work.  It's nice to have, but I don't have strong feelings about it.